pkgs/development/python-modules/boto3/default.nix

```diff
@@ -13,11 +13,11 @@
 
 buildPythonPackage rec {
   pname = "boto3";
-  version = "1.18.54"; # N.B: if you change this, change botocore and awscli to a matching version
+  version = "1.20.13"; # N.B: if you change this, change botocore and awscli to a matching version
```

Comment:
Should we also update awscli :thinking: ?

Reply:
probably, I'll test that next

Reply:
fixed